lib/buildInEnums.js

```diff
@@ -0,0 +1,167 @@
+export const predefinedActions = {
+  androidCopyImages: 'android/copyImages',
+  copyAssets: 'copy_assets',
+};
+
+export const logWarningLevels = {
+  warn: 'warn',
+  error: 'error',
+  disabled: 'disabled',
+};
+
+export const logVerbosityLevels = {
+  default: 'default',
+  silent: 'silent',
+  verbose: 'verbose',
+};
+
+export const logBrokenReferenceLevels = {
+  throw: 'throw',
+  console: 'console',
+};
+
+export const commentStyles = {
+  short: 'short',
+  long: 'long',
+  none: 'none',
+};
+
+export const commentPositions = {
+  above: 'above',
+  inline: 'inline',
+};
+
+export const commentStylesFileHeader = {
+  short: 'short',
+  long: 'long',
+  xml: 'xml',
+};
+
+export const fileFormatNames = {
+  androidColors: 'android/colors',
+  androidDimens: 'android/dimens',
+  androidFontDimens: 'android/fontDimens',
+  androidIntegers: 'android/integers',
+  androidResources: 'android/resources',
+  androidStrings: 'android/strings',
+  composeObject: 'compose/object',
+  cssVariables: 'css/variables',
+  flutterClassDart: 'flutter/class.dart',
+  iosColorsH: 'ios/colors.h',
+  iosColorsM: 'ios/colors.m',
+  iosMacros: 'ios/macros',
+  iosPlist: 'ios/plist',
+  iosSingletonH: 'ios/singleton.h',
+  iosSingletonM: 'ios/singleton.m',
+  iosStaticH: 'ios/static.h',
+  iosStaticM: 'ios/static.m',
+  iosStringsH: 'ios/strings.h',
+  iosStringsM: 'ios/strings.m',
+  iosSwiftAnySwift: 'ios-swift/any.swift',
+  iosSwiftClassSwift: 'ios-swift/class.swift',
+  iosSwiftEnumSwift: 'ios-swift/enum.swift',
+  javascriptEs6: 'javascript/es6',
+  javascriptModule: 'javascript/module',
+  javascriptModuleFlat: 'javascript/module-flat',
+  javascriptObject: 'javascript/object',
+  javascriptUmd: 'javascript/umd',
+  json: 'json',
+  lessIcons: 'less/icons',
+  lessVariables: 'less/variables',
+  scssIcons: 'scss/icons',
+  scssMapDeep: 'scss/map-deep',
+  scssMapFlat: 'scss/map-flat',
+  scssVariables: 'scss/variables',
+  stylusVariables: 'stylus/variables',
+  typescriptEs6Declarations: 'typescript/es6-declarations',
+  typescriptModuleDeclarations: 'typescript/module-declarations',
+};
+
+export const predefinedTransformGroupNames = {
```

Comment:
```suggestion
export const transformGroups = {
```

Reply:
Same here, what if you ever have an array of actual transformGroup objects in your code?

Reply:
see https://github.com/amzn/style-dictionary/pull/1375#discussion_r1819115164